requirements/requirements-app.txt

```diff
@@ -1,62 +1,38 @@
-appdirs==1.4.3
-asyncpg==0.18.3
+asyncpg==0.21.*
 atomicwrites==1.3.0
-attrs==19.1.0
-boto3==1.9.224
-botocore==1.12.224
-certifi==2019.6.16
-chardet==3.0.4
+attrs==20.*
+boto3==1.16.*
+certifi==2020.12.5
 dataclasses-json==0.5.*
-django_cte==1.1.*
 ddtrace==0.37.1
```

Comment:
This version doesn't work on (my?) Win10; is there a reason not to bump it to 0.40?

Reply:
@kwhickey @bminahan-kc @gpontejos Any input on a newer version of DDtrace?

Reply:
.40 should be fine. The latest is 0.45 if we wanna go there. We use dd-agent version 6.15 currently which looks compatible.

Reply:
Changed to 0.45.x